scripts/github_pr/github_pr_cloud.yaml

```diff
@@ -66,6 +71,27 @@ template:
               job_cmd: "../jenkins/jenkins-job-trigger"
               job_parameters:
                 *mkcloud_automation_parameters
+      - type: FileMatch
+        config:
+          paths:
+            -  !ruby/regexp '/scripts\/jenkins\/ardana\//'
```

Comment:
This won't work as the previous FileMatch filter does not let these through. For now I'd suggest you combine these into one filter invocation, with the downside of always executing both crowbar and ardana even when one would be sufficient.
Then implement the following feature for github-pr to be able to split them up again:
```
- type: EagerOrFilters
  config:
    filters:
      - type: FileMatch
        ...
      - type: FileMatch
        ...
  blacklist_handler:
    - type: SetStatus
      parameters:
        status: success
        message: no gating applicable
  whitelist_handler:
    - type: SetStatus
      parameters:
        status: pending
        message: Queued automation PR gating
```
Or maybe there is a better way?

Reply:
So there's no sort of fall-through for the FileMatch? In https://github.com/SUSE-Cloud/automation/pull/2640/files there are multiple FileMatch sections, but that may be a bad example I'm following.

Reply:
That is a good solution.